packages/calypso-e2e/src/browser-manager.ts

```diff
@@ -4,60 +4,76 @@
  */
 
```

Comment:
With all these great re-usable functions, is it worth adding some stuff to the readme about when test writers should use some of these? (E.g. when to make a new context/page).

Optional, and definitely doesn't have to be done on this PR!

Reply:
Sure, I can take a stab at expanding the documentation some more.

Reply:
In the interest of not blocking your flakey test PR, I will do this in a separate PR.